### PR TITLE
Edit UG to debug issues

### PR DESCRIPTION
**Pull request by AidenLYT**
Edited UG to reflect bugs pointed out for testers during PE-D

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ So say goodbye to traditional address book applications and say hello to the fut
 > * LookMeUp text field supports **command history** accessibility.
     >   * You can make use of your `Up` and `Down` arrow keys to navigate through the commands that you have previously entered.
 
+> [! WARNING]
+> Command prefixes (n/…​ a/…​ p/…​ e/…​ t/…​) only accepts lower case characters.
+>   * Examples like N/…​ A/…​ P/…​ E/…​ T/…​ where prefixes are capital letters will not be accepted.
+
```
Comment (smolegz) [inline docs/UserGuide.md:73]: Nice to include command constraints.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -85,7 +89,20 @@ Adds a person to the address book.
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 > [!TIP]
-> A person can have any number of tags (including 0)
+> 
+> * A person can have any number of tags **(including 0)**.
+> 
+> * There are no character limit restrictions for each input.
+>   * However, it is advisable to keep each field under **100 characters** to ensure compatibility with your device's resolution.
```
Comment (smolegz) [inline docs/UserGuide.md:96]: Good point

File: docs/UserGuide.md [hunk 3]
```diff
@@ -129,6 +146,15 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
+> [! NOTE]
```
Comment (smolegz) [inline docs/UserGuide.md:149]: Please fix this, should be `[!Note]`

**Review by smolegz — CHANGES_REQUESTED** [review]
Aiden, just a minor edit, otherwise LGTM!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -19,11 +19,11 @@ So say goodbye to traditional address book applications and say hello to the fut
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/AY2324S2-CS2103T-T12-2/tp/releases/tag/v1.3(final)).
+1. Download the latest `LookMeUp.jar` from [here](https://github.com/AY2324S2-CS2103T-T12-2/tp/releases/tag/v1.3(final)).
 
 1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar LookMeUp.jar` command to run the application.<br>
```
Comment (smolegz) [inline docs/UserGuide.md:26]: I think it'll be better to change to the format (as shown in picture below) that we'll submit for v1.4:
https://nus-cs2103-ay2324s2.github.io/website/schedule/week13/project.html
image

**Review by smolegz — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (AidenLYT) [reply docs/UserGuide.md]: Keen eye! Thank you for spotting the error!

**Pull request merged**